fea-rs/src/compile/compile_ctx.rs

```diff
@@ -331,6 +333,10 @@ impl<'a, F: FeatureProvider, V: VariationInfo> CompilationCtx<'a, F, V> {
             gdef.mark_glyph_sets = sorted.into_iter().map(|(_, cls)| cls).collect();
         }
 
+        if gdef.ligature_pos.is_empty() {
+            gdef.ligature_pos = user_lig_carets;
+        }
```

Comment:
else ... warn if the user had specified carets? Or comment to explain why just ignoring the user carets is fine.

Reply:
oops this was just a bad name, the `user_lig_carets` are the ones generated by the compiler.